includes/admin/class-wc-payments-admin.php

```diff
@@ -909,6 +909,8 @@ public function add_disputes_notification_badge() {
 
 		foreach ( $submenu[ self::PAYMENTS_SUBMENU_SLUG ] as $index => $menu_item ) {
 			if ( 'wc-admin&path=/payments/disputes' === $menu_item[2] ) {
+				// Direct the user to the disputes which need a response by default.
+				$submenu[ self::PAYMENTS_SUBMENU_SLUG ][ $index ][2]  = admin_url( add_query_arg( [ 'filter' => 'awaiting_response' ], 'admin.php?page=' . $menu_item[2] ) ); // phpcs:ignore WordPress.WP.GlobalVariablesOverride.Prohibited
```

Comment:
This is a somewhat hacky way to make sure the user is directed to the disputes which need a response by default. 

If I tried to change [this line](https://github.com/Automattic/woocommerce-payments/blob/4f5e56684535b43dc23ad1c89b9bf9506b381e6f/includes/admin/class-wc-payments-admin.php#L135) similar to what Eric mentioned here (p1655786935805679-slack-C03LA109BKM), it wouldn’t work and would error with “you can't access that page” kind of errors (see below). 

<img width="798" alt="Screen Shot 2022-06-27 at 5 35 29 pm" src="https://user-images.githubusercontent.com/8490476/175885264-06cf1f18-6253-438b-8268-6b9a29ad86ad.png">

I discovered that WCPayments has to add query args for the settings item and it has a comment about changing the submenu item global being the only way to have an absolute URL [here](https://github.com/Automattic/woocommerce-payments/blob/4f5e56684535b43dc23ad1c89b9bf9506b381e6f/includes/admin/class-wc-payments-admin.php#L317).

This line essentially does the same by altering the submenu global so that the disputes url includes the `awaiting_response` filter query arg.

Reply:
> If I tried to change [this line](https://github.com/Automattic/woocommerce-payments/blob/4f5e56684535b43dc23ad1c89b9bf9506b381e6f/includes/admin/class-wc-payments-admin.php#L135) similar to what Eric mentioned here (p1655786935805679-slack-C03LA109BKM), it wouldn’t work and would error with “you can't access that page” kind of errors.

Dang, that's unfortunate. Should we check in with wc-admin / isotope team about the best way to do things like this? Maybe the wc-admin API could be improved, or maybe we can better understand why it doesn't work (i.e. are we misusing the system).